client/src/elm/Pages/Prenatal/Activity/Utils.elm

```diff
@@ -315,6 +354,14 @@ activityCompleted currentDate assembled activity =
             (not <| expectActivity currentDate assembled PrenatalImmunisation)
                 || List.all (immunisationTaskCompleted currentDate assembled) immunisationTasks
 
+        Backend.PrenatalActivity.Model.Breastfeeding ->
+            -- @todo
```

Comment:
Anything to write on this todo? or it doesn't matter?

Reply:
(Here and below)

Reply:
These are just placehoders for activities that will be implemented in follow up PRs

Reply:
👍